### ISSUE TITLE
Share your new ideas!

### ISSUE THREAD

**Issue by ghost** [theupweb/idea-lab#3]
All the contributors of Devscript Winter of Code and others can share their ideas "IdeaLab project".
We Strongly support Opensource and uplifting communities!

**Comment by ghost** [comment]
@JoshuaPoddoku 
Hi 
I was thinking that we can make search for developer according to their client location so that if client want to meet  developer in person they can easily arrange meeting.

**Comment by ankita413** [comment]
We can add filter by options on the basis of tech stack like react,node, MERN, MEAN etc

**Comment by ankita413** [comment]
adding filter by option the user will have more clear and concise search results

**Comment by ghost** [comment]
@ankita413, great inputs! We can take this forward.

**Comment by ghost** [comment]
We can add news section that will provide developer about new libraries ,research paper and language upgrade.

**Comment by Priyanshu-official** [comment]
@niketbanetha I would like to add Open-source Hackathon Update 
In one Place we can have more option No need to go here and there.
also we can **add Push Notification feature for Upcoming  Hackathon**.
@JoshuaPoddoku  **What you say about this? is it good or not...**

**Comment by ghost** [comment]
@Priyanshu-official yes we can also add about new opportunities or project in which developer can enroll themselves.

**Comment by ghost** [comment]
Yes, you can add a push notification feature for updates on a topic which a user follows.

Thanks for the inputs @niketbanetha and @Priyanshu-official!

**Comment by ghost** [comment]
Join this workspace: https://app.zenhub.com/workspaces/idea-lab-project-5fd66ab0e0b3370014153707/board

**Comment by Priyanshu-official** [comment]
@JoshuaPoddoku When we are going to start Making this Project?

**Comment by ghost** [comment]
According to the schedule it is 25 Dec

**Comment by Priyanshu-official** [comment]
@JoshuaPoddoku  Ok thanks

**Comment by himanshu007-creator** [comment]
HI 
I had a suggestion. Developers sometimes face a problem, they want a person, who can review their work before releasing it to clients. We can create a platform where developers with similar experience can help reviewing each other's work.

**Comment by ghost** [comment]
This is a great add-on @himanshu007-creator. Looking forward to having you as a contributor

**Comment by himanshu007-creator** [comment]
thanks a lot!

**Comment by Rimpi246** [comment]
May be something like a post(sneak peek) to show what a developer is currently working on and then designers can see and decide if  they want to collaborate.

**Comment by suhas-kadu** [comment]
We can add a Blogs section where some seniors or Alumni can share their experience regarding some top events like GSOC and other competitions.

**Comment by Rimpi246** [comment]
And there should be private communicating platform

**Comment by ankita413** [comment]
@Rimpi246 u mean to say that they can chat or something?

**Comment by suhas-kadu** [comment]
I think we can add a section for linked url of developer in his profile, for private communication.

**Comment by Rimpi246** [comment]
> @Rimpi246 u mean to say that they can chat or something?

Yeah

**Comment by Vrushti24** [comment]
sir #41  please check that
